pkg/api/v1beta3/types.go

```diff
@@ -1691,7 +1691,7 @@ type ComponentCondition struct {
 // ComponentStatus (and ComponentStatusList) holds the cluster validation info.
 type ComponentStatus struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"standard object metadata; see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#metadata"`
 
 	Name       string               `json:"name,omitempty" description:"name of the component"`
```

Comment:
Yeah, that's an omission. I removed from the internal, but not here. Will remove it.